src/ralph/schemas/edx/converters/xapi/server_event_to_xapi.py

```diff
@@ -0,0 +1,65 @@
+"""Server event xAPI Converter"""
+
+from ralph.schemas.edx.converters.base_converter import GetFromField
+from ralph.schemas.edx.server_event import ServerEventSchema
+
+from . import constants as const
+from .base import BaseXapiConverter
+
+
+class ServerEventToXapi(BaseXapiConverter):
+    """Converts a common edx server event to xAPI
+    See ServerEventSchema for info about the Edx server event
+    Example Statement: John viewed https://www.fun-mooc.fr/ Web page
+    """
+
+    _schema = ServerEventSchema()
+
+    conversion_dict = {
+        "version": const.VERSION,
+        "actor": {
+            "account": {
+                "name": GetFromField(
+                    "username", lambda username: username if username else "anonymous"
+                ),
+                "homePage": lambda: BaseXapiConverter._platform,
+            },
+            "objectType": "Agent",
+        },
+        "verb": {"id": const.XAPI_VERB_VIEWED, "display": {"en": const.VIEWED}},
+        "object": {
+            "id": GetFromField(
+                "event_type",
+                lambda event_type: BaseXapiConverter._platform + event_type,
+            ),
+            "definition": {
+                "type": const.XAPI_ACTIVITY_PAGE,
+                "name": {"en": const.PAGE},
+            },
+            "objectType": "Activity",
+        },
+        "context": {
+            "platform": lambda: BaseXapiConverter._platform,
+            "extensions": {
+                const.XAPI_EXTENSION_ACCEPT_LANGUAGE: GetFromField("accept_language"),
+                const.XAPI_EXTENSION_AGENT: GetFromField("agent"),
+                const.XAPI_EXTENSION_COURSE_ID: GetFromField("context>course_id"),
+                const.XAPI_EXTENSION_COURSE_USER_TAGS: GetFromField(
+                    "context>course_user_tags",
+                    lambda course_user_tags: course_user_tags
+                    if course_user_tags
+                    else {},
+                ),
```

Comment:
One question, the [xAPI spec](https://github.com/adlnet/xAPI-Spec/blob/1.0.3/xAPI-Data.md#requirements) states `Statements and other objects SHOULD NOT include properties with a value of an empty object.` 

What should we do if course_user_tags is not present (or equal to an empty object) ? It seems we can't set it to {} by default